components/uponor_smatrix.rst

```diff
@@ -0,0 +1,150 @@
+Uponor Smatrix Base Pulse Underfloor Heating
+============================================
+
+.. seo::
+    :description: Instructions for setting up an Uponor Smatrix Base Pulse underfloor heating control system in ESPHome.
+    :keywords: Uponor Smatrix, HCS, Thermostat
+
+The Uponor Smatrix component allows you to integrate an Uponor Smatrix Base Pulse underfloor heating control system in ESPHome without the need for an Smatrix Pulse Com R-208 communication module.
+It directly communicates with the controller and thermostats via the RS485 thermostat bus.
+
+Connecting to the bus
+---------------------
+
+This component is able to communicate directly with the RS485 thermostat bus. For that, you will need to connect an RS485 to TTL converter to a UART bus of your ESPHome device.
+
+The RS485 side of the converter can either be connected to one of the A/B terminals on the controller or on one of the thermostats.
+The +/- terminals provide 5 volts and can be used to power your ESPHome device.
+
+The :ref:`UART Component <uart>` must be configured with a baud rate of 19200, 8 data bits, no parity, 1 stop bit.
+
+.. _uponor-gettingstarted:
+
+Getting started
+---------------
+
+The controller and the thermostats have unique addresses used for communication that are not displayed anywhere but can only be found when scanning the bus.
+Start with a basic configuration that just contains the UART and Uponor hub components. Make sure that the UART pins are configured according to your wiring.
+
+.. code-block:: yaml
+
+    uart:
+      baud_rate: 19200
+      tx_pin: GPIO17
+      rx_pin: GPIO16
```

Comment:
Please don't add `uart` blocks in examples, users need to read the above docs pointint to uart and set it up according to their own pins etc and the required baud rate.
```suggestion
```